Formula/minimesos.rb

```diff
@@ -0,0 +1,16 @@
+class Minimesos < Formula
+  desc "Testing infrastructure for Mesos frameworks"
+  homepage "https://minimesos.org/"
+  url "https://github.com/ContainerSolutions/minimesos/archive/0.10.2.tar.gz"
+  sha256 "16b198078c9384426f8849b08e92e173816c4d6626331bc6569aa5de89dffbdd"
+
+  depends_on "docker"
+
+  def install
+    bin.install "bin/minimesos"
+  end
+
+  test do
+    system "test -e #{bin}/minimesos && true || false"
```

Comment:
Apologies, I ran into issues trying that, minimesos requires not only the docker client but the docker server running as well. The `depends_on "docker"` only installs the client which is useless without the server. I tried going the docker-machine virtualbox route but that required virtualbox which is in homebrew-cask and which doesn't play nice within the confines of the test environment. Lastly I tried docker-machine-xhyve but that required root permission which wasn't a viable option either. Not sure what my next steps should be, should this be in homebrew-cask or is there another option?